code/src/layouts/app-products/AppProductsList.scss

```diff
@@ -0,0 +1,38 @@
+@import "@design-system";
+
+.spa-products-list {
+  margin-top: spacing(5);
+  padding: spacing(12) spacing(12);
+  background-color: var(--spa-foreground-background-color, $white);
+  display: flex;
+  flex-direction: column;
+  align-items: stretch;
```

Comment:
I hope it is default value, so we don't need it

Reply:
Moved